examples/running_mean/standard_scaler/optimizer.rs

```diff
@@ -0,0 +1,93 @@
+use crate::standard_scaler::{Config, Input, Output, ScalingError, StandardScaler};
+use linfa::{Fit, IncrementalFit};
+use ndarray::{Axis, Data};
+
+/// It keeps track of the number of samples seen so far, to allow for
+/// incremental computation of mean and standard deviation.
+pub struct OnlineOptimizer {
+    pub n_samples: u64,
+}
+
+/// Initialize n_samples to 0.
+impl Default for OnlineOptimizer {
+    fn default() -> Self {
+        Self { n_samples: 0 }
+    }
+}
+
+impl<S> Fit<Config, Input<S>, Output> for OnlineOptimizer
+where
+    S: Data<Elem = f64>,
+{
+    type Error = ScalingError;
+
+    fn fit(
+        &mut self,
+        inputs: &Input<S>,
+        _targets: &Output,
+        blueprint: Config,
```

Comment:
why not call this `config` or `params`?  Is there any other ML library that uses the "blueprint" vocabulary?

`params` as in scikit-learn would be more accurate IMO -- we are not providing model configuration but model parameters.

Reply:
Params is kind of an overloaded term: in this case, I'd say that we are passing hyperparameters (e.g. number of convolutional layers in a CNN), not parameters (e.g. the network weights).
I think it's quite natural to call the set of model hyperparameters `model configuration`. 
We can safely discard the blueprint terminology, but I'd try to stick to terms that are not ambiguous.

Reply:
Why do you prefer "model configuration" to "hyperparameter" for that purpose?